test/BindTest.php

```diff
@@ -64,7 +64,7 @@ public function setUp()
         if (getenv('TESTS_ZEND_LDAP_ALT_USERNAME')) {
             $this->altUsername = getenv('TESTS_ZEND_LDAP_ALT_USERNAME');
         }
-        $this->principalName = getenv('TESTS_ZEND_LDAP_PRINCIPAL_NAME');
+        $this->altPrincipalName = getenv('TESTS_ZEND_LDAP_ALT_PRINCIPAL_NAME');
```

Comment:
@heiglandreas I had to change this because the Manager use is not searchable with the search filter